docs/how_we_work/estimation.md

```diff
@@ -0,0 +1,18 @@
+# Estimation
+
+As a team, we will use points to estimate our issues so that we can better plan our scope for each cycle and our overall roadmap.
+
+## How we'll estimate
+
+We will use points for estimation, with an exponential scale: 1, 2, 4, 8, 16, 32.
+
+Reference points for what each level means, via [@amymok](https://github.com/18F/OPRE-Unicorn/issues/77):
+
+Points | Meaning
+---|---
+1 | Really trivial. Typos, word changes.
+2 | Easy to change. No need to do additional research. Requires a bit more effort than a trivial task. Examples: Fix a linter error. Fix a single unit test. Reword paragraphs. 
+4 | Requires more effort. A medium task. May some research on reminding yourself how to do a task, but no unknowns.
```

Comment:
Typo fix.  Change "May some research" to "May involve some research" or "May require some research"

Reply:
Good catch, thanks @ninamak!